conda_build.recipe/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: numpy
+  version: 1.11.0{{ '+' + environ.get('GIT_DESCRIBE_NUMBER', '0') + '.' + environ.get('GIT_DESCRIBE_HASH', '0') if environ.get('GIT_DESCRIBE_NUMBER', '0') != '0' else "" }}
```

Comment:
> As for the conda NumPy ABI, there is nothing special that needs to be done to the recipe to get it to track the NumPy version

Right. My concern is that for dev releases, like a 1.11 prerelease, we actually don't want abi tracking, because that will require all packages in the environment to have been built against the 1.11-final-abi, and neither the packages nor even the abi will be available while 1.11 is still in development. Does that make sense?